frontend/src/components/pages/CardDetails/Presentation.jsx

```diff
@@ -19,33 +19,41 @@ import AssigneesList from "../../widgets/AssigneesList";
 import ReviewersTable from "../../widgets/ReviewersTable";
 import Reviews from "./Reviews";
 
-const useStyles = makeStyles((theme) => {
-  return {
-    paper: {
-      padding: theme.spacing(1, 2, 1),
-    },
+const TableCell = withStyles({
+  root: {
+    borderBottom: "none",
+    padding: "3.24%",
+  },
+})(MuiTableCell);
 
-    commentColumn: {
-      minWidth: 300,
+const useStyles = makeStyles((theme) =>
+  createStyles({
+    root: {
+      flexGrow: 1,
     },
-
-    tableContainer: {
-      maxHeight: 200,
+    paper: {
+      padding: theme.spacing(2),
+      height: "100%",
+      textAlign: "auto",
+      boxSizing: "border-box",
+      maxHeight: 350,
```

Comment:
The mix 'n match height styling is odd. In this file you have maxHeight: 350 (why have a max height anyway?)  and then in another file you have:

```
table: {
      height: "20vh",
      [theme.breakpoints.down("md")]: {
        maxHeight: "30vh",
      },
    },
```

And these things are supposed to sit side by side. It's very odd.

As soon as I comment out the heights then the weird scrolling behavior goes away.

Reply:
The scrolling behavior was there for a reason. So that we display a max number of reviewers. But I will take it out.